examples/motor-imagery/plot_frequency_band_selection.py

```diff
@@ -0,0 +1,223 @@
+                               verbose='warning')
+
+events, _ = events_from_annotations(raw_filter, event_id,
+                                    verbose='warning')
+
+# Read epochs (train will be done only between 0.5 and 2.5 s)
+epochs = Epochs(
+    raw_filter,
+    events,
+    event_id,
+    tmin,
+    tmax,
+    proj=True,
+    picks=picks,
+    baseline=None,
+    preload=True,
+    verbose=False)
+labels = epochs.events[:, -1] - 2
+
+# Get epochs
+epochs_data_baseline = 1e6 * epochs.get_data()
+
+# Compute covariance matrices
+cov_data_baseline = Covariances().transform(epochs_data_baseline)
+
+# Split data into training and test set
+indices = np.array(range(labels.shape[0]))
+Cov_train, Cov_test, y_train, y_test, train_ind, test_ind = \
+    train_test_split(cov_data_baseline, labels, indices, test_size=0.2,
+                     random_state=42)
+
+# Set classifier
+model = MDM(metric=dict(mean='riemann', distance='riemann'))
+
+# Classification with minimum distance to mean
+model.fit(Cov_train, y_train)
+acc_baseline = model.score(Cov_test, y_test)
+t1 = time() - t0
+
+###############################################################################
+# Pipeline with a frequency band selection based on the class distinctiveness
+# ----------------------------------------------------------------------------
+#
+# Step1: Select frequency band maximizing class distinctiveness on
+# training set.
+#
+# Define parameters for frequency band selection
+t2 = time()
+freq_band = [5., 35.]
+sub_band_width = 4.
+sub_band_step = 2.
+alpha = 0.4
+
+# Select frequency band using training set
+best_freq, all_class_dis = \
+    freq_selection_class_dis(raw, freq_band, sub_band_width,
+                             sub_band_step, alpha,
+                             tmin, tmax,
+                             picks, event_id,
+                             train_ind=train_ind, method='train_test_split',
+                             return_class_dis=True, verbose=False)
+
+print('Selected frequency band : ' + str(best_freq[0])
+      + '-' + str(best_freq[1]) + ' Hz')
+
+###############################################################################
+# Step2: Train classifier using selected frequency band and evaluate
+# performance using test set
+
+# Apply band-pass filter using the best frequency band
+best_raw_filter = raw.copy().filter(best_freq[0], best_freq[1],
+                                    method='iir', picks=picks,
+                                    verbose=False)
+
+events, _ = events_from_annotations(best_raw_filter, event_id,
+                                    verbose=False)
+
+# Read epochs (train will be done only between 0.5 and 2.5s)
+epochs = Epochs(
+    best_raw_filter,
+    events,
+    event_id,
+    tmin,
+    tmax,
+    proj=True,
+    picks=picks,
+    baseline=None,
+    preload=True,
+    verbose=False)
+
+# Get epochs
+epochs_data_train = 1e6 * epochs.get_data()
```

Comment:
use units

Reply:
Okay, I modified all `epochs.get_data` that way.